.gitignore

```diff
@@ -1 +1,2 @@
 _site/
+/.idea
```

Comment:
This folder is for application-specific files to ignore and your editor-specific extensions should not pollute `.gitignore`. Can you instead add it to your [local global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)?